AirCasting/Utils/Textfield.swift

```diff
@@ -7,27 +7,27 @@
 
 import SwiftUI
 
-func createTextfield(placeholder: String, binding: Binding<String>) -> some View {
+func createTextfield(placeholder: String, binding: Binding<String>, shouldWarnUser: Bool = false) -> some View {
```

Comment:
I would call this argument `isInputValid`, cause I think that's the purpose of the red border (to show that input is not valid)

Reply:
Brilliant idea! Thanks!